pkg/runs/submitter.go

```diff
@@ -304,10 +306,11 @@ func (submitter *Submitter) submitRun(
 
 			var resultGroup *galasaapi.TestRuns
 			resultGroup, err = submitter.launcher.SubmitTestRun(groupName, className, requestType, requestor,
-				nextRun.Stream, nextRun.Obr, trace, submitOverrides)
+				nextRun.Stream, nextRun.Obr, trace, nextRun.GherkinUrl, nextRun.GherkinFeature, submitOverrides)
 			if err != nil {
 				log.Printf("Failed to submit test %v/%v - %v\n", nextRun.Bundle, nextRun.Class, err)
 				lostRuns[className] = &nextRun
+				//This is thrown when test submit is run.... Unit test and fix
```

Comment:
What does this mean?

Reply:
Sorry this is a note for myself to where I found an error when doing submission tests. I have since fixed and united tested. I will remove it